Makefile

```diff
@@ -237,7 +249,7 @@ $(build_dir)/%.d : $(src_dir)/%.[c,S]
 
 $(objs-y):
 	@echo "Compiling source	$(patsubst $(cur_dir)/%, %, $<)"
-	@$(cc) $(CFLAGS) -c $< -o $@
+	-@$(cc) $(CFLAGS) -c $< -o $@
```

Comment:
Should remove '-'

Reply:
Good catch. Fixed in commit 83baede9561218058ae4cfd5af469e84fc96ad7c